pkg/storage/strategies/strategy.go

```diff
@@ -0,0 +1,57 @@
+package strategies
+
+import (
+	"fmt"
+)
+
+// Strategy is an enum denoting the database strategy. Implements the pflag.Value interface.
+//
+//	type Value interface {
+//		String() string
+//		Set(string) error
+//		Type() string
+//	}
+type Strategy struct {
+	name string
+}
+
+var (
+	Postgres = Strategy{name: "postgres"}
+	Off      = Strategy{name: "off"}
```

Comment:
Redis should be another strategy here?

Reply:
This is strategies related to a backing store. Redis and lru, for example could be cache strategies.

Reply:
I am a little confused. My understanding is that there is a store where we maintain the counters of the received data. This store can be redis, postgres or other impl. What is this referred to as? I know there is a postgres store where we store the limits from UI and tenant details

Reply:
This is the backing store for all configurations, keys and tenants which will always be postgres.

The storage of counters can be postgres or redis. Depends on the policy selected in the `signozlimiter` processor.

Reply:
Thanks, I understand it now.